.template/addons/docker/Dockerfile.tt

```diff
@@ -120,6 +120,12 @@ RUN yarn install --network-timeout 100000
 COPY . ./
 
 <%- if WEB_VARIANT -%>
```

Comment:
Why are these ARG defines so low in the file and not with the other ARG on line 5-8?